### PR TITLE
mkcloud: dont source mkcloud.config

### PR DESCRIPTION
**Pull request by bmwiedemann**
we actually want to provide our config from the outside
and qa_crowbarsetup now sources mkcloud.config for us
which would override the values we wanted

### CODE REVIEW

**Comment by jsuchome** [comment]
What about adding comment (like the one in commit message) to the line?

**Comment by bmwiedemann** [comment]
added a comment

File: scripts/mkcloud [hunk 1]
```diff
@@ -50,6 +50,7 @@ mkcloud_temporary_scripts="mkcloud-libvirt.sh"
 for script in $mkcloud_temporary_scripts; do
     source ${mkcloud_lib_dir}/$script
 done
+rm -f mkcloud.config # we dont want to source old info in the line below
```
Comment (dirkmueller) [inline scripts/mkcloud]: this should be 

```bash 
  rm -f $mkcconf

```

**Comment by dirkmueller** [comment]
+1

**Pull request merged**